src/spryker/docker/image/console/root/lib/task/init.sh.twig

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+function task_init()
+{
+    task "database:available"
+    task "http:wait" $JENKINS_URL
+    task "rabbitmq:vhosts"
+    passthru "vendor/bin/install -r docker -s queue-flush"
+
+    task "assets:apply" # calls task "install" when database is not applied
+
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s queue-worker"
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s jenkins-up"
```

Comment:
can move this stuff into https://github.com/inviqa/harness-base-php/blob/0.2.x/src/_base/docker/image/console/root/lib/task/init.sh.twig#L8 init steps rather than disable _base functionality

Reply:
wilco